README.md

```diff
@@ -242,23 +242,67 @@ For complete documentation of the `mp-build` tool see the [MP Build Tool Documen
 
 ### Creating a New Management Pack (Cassandra-DB)
 <details><summary>
-This guide assumes you have already set up the SDK and know how to create a new project. 
-It walks you through the steps necessary to monitor an endpoint, using Cassandra DB as 
+This guide assumes you have already set up the SDK and know how to create a new project.
+It walks you through the steps necessary to monitor an endpoint, using Cassandra DB as
 an example.</summary>
 TODO
 </details>
 
 ### Extending an Existing Management Pack (MySQL)
 <details><summary>
-This guide assumes you have already set up the SDK and know how to create a new project. 
+This guide assumes you have already set up the SDK and know how to create a new project.
 It walks you through the steps necessary to extend an existing Management Pact to add
 additional data, using the MySQL Management Pack as an example.</summary>
 TODO
 </details>
 
 
 ## Troubleshooting
-TODO
+
+If you encounter any issues while using the VMware Aria Operations Integration SDK, you can refer to the troubleshooting guides for each tool:
+
+- [mp-test Troubleshooting Guide](doc/mp-test.md#troubleshooting)
+- [mp-init Troubleshooting Guide](doc/mp-init.md#troubleshooting)
+- [mp-build Troubleshooting Guide](doc/mp-build.md#troubleshooting)
+
+### Docker Issues
+
+If you encounter any issues with Docker while using the VMware Aria Operations Integration SDK, you can refer to the following guides:
```

Comment:
This says 'any issues', but the two guides are just about getting Docker running. Like before, the text is generally pretty good but let's make sure the heading is a specific problem/question that matches the text.